streets_poles/init.lua

```diff
@@ -11,7 +11,7 @@ minetest.register_node(":streets:bigpole", {
 	drawtype = "nodebox",
 	tiles = { "streets_pole.png" },
 	sunlight_propagates = true,
-	groups = { cracky = 1, level = 2, bigpole = 1, not_blocking_trains = 1},
```

Comment:
What does `not_blocking_trains` do? Why can it be removed?

Reply:
I don't know what it's there for, I didn't add it for any reason.

Reply:
This is here for advtrains. If a block is registered with this group, trains will not be blocked even if this block is in its path. This is useful as poles are not full blocks, hence not visually blocking trains but technically would.